src/js/view/controls/rightclick.js

```diff
@@ -77,7 +83,10 @@ export default class RightClick {
         return false;
     }
 
-    hideMenu() {
+    hideMenu(evt) {
+        if (evt && evt.type === 'mousedown') {
+            return;
+        }
```

Comment:
@robwalch I added this because `mousedown` was being triggered on iOS after the right click became visible causing it to become hidden. Is there a better way to guard against that occurring?

Reply:
`mousedown` or `touchend`? There is no `touchend` event listener here. If you don't want `mousedown` to run `hideMenu` then remove `element.addEventListener('mousedown', this.hideMenuHandler);` for the player element or the document, or both. The issue there is that the menu is supposed to close the moment you begin interacting with something else on the page.

I don't think you want to filter `hideMenu` by event type, I think you need to look specifically at the targets for the situation you are describing. Please include the contents of the `mousedown` event or show me the debugger state in Safari when the menu opens. It makes sense for `mousedown` to fire when a tap starts, not when it ends.